apps/health_campaign_field_worker_app/lib/utils/i18_key_constants.dart

```diff
@@ -6,6 +6,7 @@ const forgotPassword = ForgotPassword();
 const home = Home();
 const searchBeneficiary = SearchBeneficiary();
 const householdLocation = HouseholdLocation();
+const housholdDetails = HousholdDetails();
```

Comment:
typo in householdDetails and HouseholdDetails and several references to these

Reply:
@egov-joy   resolved